backend/src/main/java/com/huellapositiva/domain/model/valueobjects/ProfileLocation.java

```diff
@@ -0,0 +1,15 @@
+package com.huellapositiva.domain.model.valueobjects;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ProfileLocation {
+    private final String province;
+    private final String town;
+    private final Integer zipCode;
+    private final String address;
```

Comment:
Missing island

Reply:
Deleted class because dont use that. We have Location as valueObject, there we will do the validation.